recipes/ibis-heavyai/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "ibis-heavyai" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ibis-heavyai-{{ version }}.tar.gz
+  sha256: ccebecc331eeb8c4f50df7984e7445bfebd25285bd674d2ec2a6326002b7ece0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - flit-core
+  run:
+    - python >=3.7
+    - ibis-framework >=2.0, <3
+    - heavyai
+    - rbc >=0.8.0
+    - pyarrow >=3.0.0
+    - regex
+    - pandas
+    - shapely
+    - geopandas
+    - importlib_metadata
+
+test:
+  imports:
+    - ibis_heavyai
```

Comment:
Please remember to add pip check back if updating to the latest version solves the pinning problem.

Reply:
We will check this for the next release sure.